### PR TITLE
Fixes some cases where changeturf were called incorrectly

### LINKED ISSUE #32519: All wires on derelict station point up
Round ID: 78491

For clarity, I mean the exploded, "Russian" station.  I spawned on one of the derelict drone shells and discovered (along with one other player) that all the wires point north.  On tiles where there was more than one wire, they were all on top of each other.

I crowbarred up a few floor tiles to take [this image](https://i.imgur.com/FDyxM59.png), in case my description sucks.

I don't recall this being a problem the last time I was on the derelict station (which I believe was last night) and the server revision points to a [commit from 3 days ago](https://github.com/tgstation/tgstation/commit/d5b9b56858e7c0c52fe1fe239e76defef1f2b180).  I'll be trying it again in a moment to see if it persists between rounds.

EDIT: Seems to have persisted to the next round.  Oddly, the round ID is now 78493.  Maybe I mistyped it originally?

EDITx2: [Extra obvious photo](https://i.imgur.com/SV0SXvo.png), because a friend didn't believe me.

EDITx3: Was poking around the recent commits and noticed [this one](https://github.com/tgstation/tgstation/commit/f6e7a7f249386d5b21179cb4ea59c488e83fee5a) from a couple days ago that was meant to clean up some wires.  It may have cleaned a bit too much.

### PR DESCRIPTION
**Pull request by ninjanomnom**
:cl: ninjanomnom
fix: Fixes ruin cable spawning and probably some other bugs
/:cl:

turf/AfterChange was being called as a result of some changeTurfs in initialize.

fixes #32519
closes #32555

### CODE REVIEW

File: code/modules/mapping/map_template.dm [hunk 1]
```diff
@@ -74,7 +74,7 @@
 	if(T.y+height > world.maxy)
 		return
 
-	var/list/bounds = maploader.load_map(file(mappath), T.x, T.y, T.z, cropMap=TRUE)
+	var/list/bounds = maploader.load_map(file(mappath), T.x, T.y, T.z, cropMap=TRUE, no_changeturf=TRUE)
```
Comment (Cyberboss) [inline code/modules/mapping/map_template.dm]: This is very very very wrong but I'm too tired to explain why

Comment (ninjanomnom) [reply code/modules/mapping/map_template.dm]: The ruins were still working fine but yeah, this likely has side effect I don't know about atm

**Comment by duncathan** [comment]
if the problem is afterchange, don't stop all calls to changeturf. i split them up for a reason

**Comment by ninjanomnom** [comment]
they aren't stopped, this is only stopping the afterchange calls

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**